js/toolbar/annotation-tool.js

```diff
@@ -24,11 +24,16 @@ function AnnotationTool(parent){
         var btnType = this.getAttribute("data-type") || "";
 
         if(_self.curType == btnType){
-            return;
+            console.log('same');
```

Comment:
Change this back to whatever it was before

Reply:
Changed

Reply:
I changed this, because it was needed for de-selecting function, i.e. when you click on the same tool, it will be de-selected and the `cursor` tool would be selected. Let me know if you want me to add this feature back.

Reply:
That makes sense. I wasn't aware that we didn't have the de-select function. So yeah, we would need a code change to be able to do deselect

Reply:
Added this back.